tests/test_basic.py

```diff
@@ -0,0 +1,223 @@
+import os           # used to get the location of the testdata
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def test_geojson_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
```

Comment:
Suggest to use `python` here, not `python3`. The actual version should be set in the environment.

Fixing `python3` here would mean that we cannot run the tests in a Python 2 environment.

Reply:
Updated.